tools/ramclustr/ramclustr.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="ramclustr" name="RAMClustR" version="@TOOL_VERSION@+galaxy1">
+    <macros>
+        <import>ramclustr_macros.xml</import>
+    </macros>
+
+    <requirements>
+        <container type="docker">recetox/ramclustr:1.1.0-recetox0</container>
+    </requirements>
+
+    <inputs>
+        <param name="input_xcms" label="input_xcms" type="data" format="RData" help=": containing grouped feature data for clustering by ramclustR" />
+        <expand macro="parameters" />
+    </inputs>
+
+    <command detect_errors="aggressive"><![CDATA[
+    
+        Rscript 
+            -e 'obj <- load("$input_xcms")'
```

Comment:
The purpose of a Galaxy tool is to make it usable by the non-IT person and to integrate it into the Galaxy ecosystem. Hence, exposing the R-specific format to the wrapper's interface is not suitable. If the input/output is tabular then expose it as a tsv (preferable) or csv files. If the tabular format contains a lot of high precision decimal numbers choose some binary tabular format (for instance h5). If the input is of some unknown structure then we have a problem and we should have a discussion on this.

Reply:
The tool itself takes an XCMS R object as input, that wasn't my choice. It can be directly wired in from XCMS in Galaxy.

Reply:
Ok, therefore, I suggest having multiple input/output types. You can specify mulltiple formats by `<param type="data" format="rdata,tsv">` and handle these formats differently in the <command> segment. If the input is only a table (or several tables) it's worth the shot.